packages/shared/friends/sagas.ts

```diff
@@ -389,18 +389,18 @@ function* configureMatrixClient(action: SetMatrixClient) {
     handleIncomingFriendshipUpdateStatus(FriendshipAction.REJECTED, socialId)
   )
 
+  client.onChannelMembers((conversation) => {
+    updateChannelInfo(conversation, client)
```

Comment:
Shouldn't we also send the update about the members list with the `UpdateChannelMembers` message? I think that's the message that will solve the blank member list and toggle issue.